### PR TITLE
feat: 139 classify menu item

### PR DESCRIPTION
**Pull request by TheLLspectre**
Suggest to classify menu item. 

- closes also #142

### CODE REVIEW

**Comment by qvaleroo** [comment]
Can we just rename `3d` by `3D` here:
image
Then OK for me 👍

**Comment by TheLLspectre** [comment]
> Can we just rename `3d` by `3D` here: image Then OK for me 👍

Done

**Comment by qvaleroo** [comment]
QA OK 👍

**Comment by Morgan-6Freedom** [comment]
can we rename `Builder` to `Builders` and `Editor` to `Editors` ?
And also, if @qvaleroo you confirm that the API pricing window is outdated, can we remove it ? 

Thank you

**Review by Morgan-6Freedom — CHANGES_REQUESTED** [review]
can we rename Builder to Builders and Editor to Editors ?
And also, if @qvaleroo you confirm that the API pricing window is outdated, can we remove it ?

Thank you

**Comment by qvaleroo** [comment]
We can remove pricing page yes 👍

**Comment by TheLLspectre** [comment]
I create a redirection to the website on API Pricing button.

File: package/Editor/Plugins/APIPricing.cs [hunk 1]
```diff
@@ -5,98 +5,10 @@ namespace Scenario.Editor
...
-            EditorGUILayout.BeginVertical(GUI.skin.box);
-            for (int i = 0; i < additionalDetails.GetLength(0); i++)
-            {
-                EditorGUILayout.BeginHorizontal();
-                GUILayout.Label(additionalDetails[i, 0], GUILayout.Width(200));
-                GUILayout.Label(additionalDetails[i, 1]);
-                EditorGUILayout.EndHorizontal();
-            }
-            EditorGUILayout.EndVertical();
+            Application.OpenURL("https://www.scenario.com/api-pricing");
```
Comment (qvaleroo) [inline package/Editor/Plugins/APIPricing.cs:11]: Better to redirect to: https://docs.scenario.com/page/api-pricing

Comment (TheLLspectre) [reply package/Editor/Plugins/APIPricing.cs]: done

**Review by Morgan-6Freedom — APPROVED** [review]

**Pull request merged**